cli/cook/subcommands/kill.py

```diff
@@ -48,51 +48,62 @@ def add(uuid, entity_type, cluster):
         raise Exception(message)
 
 
+def __kill_entities(cluster, uuids, endpoint, param, entity_type):
+    """Attempts to kill the jobs / instances / groups with the given UUIDs on the given cluster"""
+    success_status_code = 204
+    cluster_name = cluster['name']
+    resp = http.delete(cluster, endpoint, params={param: uuids})
+    if resp.status_code == success_status_code:
+        for uuid in uuids:
+            print_info(f'Killed {entity_type} {colors.bold(uuid)} on {colors.bold(cluster_name)}.')
+        return True
+    else:
```

Comment:
Should we include the response content as explanation of why the kill failed? Or at least log it?

Reply:
The response is logged in `http.delete`:

```python
def delete(cluster, endpoint, params):
    """DELETEs data corresponding to the given params on cluster at /endpoint"""
    url = __make_url(cluster, endpoint)
    resp = __delete(url, params)
    logging.info(f'DELETE response: {resp.text}')
    return resp
```